ci/kokoro/windows/builds/quickstart-bazel.ps1

```diff
@@ -45,12 +45,13 @@ function Get-Released-Quickstarts {
         # The following quickstarts have problems building on Windows:
         #   TODO(#8145) - asset (TRUE/FALSE macros)
         #   TODO(#8125) - channel (DOMAIN macro)
+        #   TODO(#11925) - compute is too big for windows
         #   TODO(#10737) - dialogflow_es triggers bug in Bazel 6.0.0
         # In addition, the `google/cloud/debugger/quickstart` directory is gone.
         # However, we use the previous release of `google-cloud-cpp` in these
         # builds, where the target still exists:
-        #    TODO(#11772) - deprecated library.
-        Where-Object { -not ("asset", "channel", "dialogflow_es", "debugger" -contains $_) } |
+        #   TODO(#11772) - deprecated library.
```

Comment:
nit: (unrelated to your PR) while we are here, maybe we should say: "debugger is a deprecated library"